src/main/resources/plugin-descriptor.properties

```diff
@@ -0,0 +1,39 @@
+# Elasticsearch plugin descriptor file
+# This file must exist as 'plugin-descriptor.properties' inside a plugin.
+#
+### example plugin for "foo"
+#
+# foo.zip <-- zip file for the plugin, with this structure:
+# |____   <arbitrary name1>.jar <-- classes, resources, dependencies
+# |____   <arbitrary nameN>.jar <-- any number of jars
+# |____   plugin-descriptor.properties <-- example contents below:
+#
+# classname=foo.bar.BazPlugin
+# description=My cool plugin
+# version=6.0
+# elasticsearch.version=6.0
+# java.version=1.8
+#
+### mandatory elements for all plugins:
+#
+# 'description': simple summary of the plugin
+description=Aiven Encrypted Storage Plugin
+#
+# 'version': plugin's version
+version=${version}
+#
+# 'name': the plugin name
+name=aiven-encrypted-storage
+#
+# 'classname': the name of the class to load, fully-qualified.
+classname=io.aiven.elasticsearch.gcs.GcsStoragePlugin
```

Comment:
We need to rename it into `io.aiven.elasticsearch.repositories.gcs.GcsRepositoryPlugin`, but let's make it a separate PR, as it will change the whole file structure.

Reply:
ok